solution_runner/commands/defaults_and_choices.py

```diff
@@ -0,0 +1,15 @@
+from datetime import datetime
+
+import consts
+
+
+def get_default_year() -> int:
+    """
+    :return: default year which is the current year if it's December, last year otherwise
```

Comment:
I like the previous docstring ("If it not December, chances are we are solving last year's challenges").